R/reactive_values.R

```diff
@@ -1,40 +1,40 @@
 setup_reactive_values_obj <- function(input) {
-    
+
     get_filename <- function(in_file) {
         infile <- in_file
         if (is.null(infile)) {
             return(NULL)
         }
         stringi::stri_extract_first(str = infile$name, regex = ".*")
     }
-    
+
     load_data <- function(in_file, two_datasets=NULL) {
         infile <- in_file
         if (is.null(infile) || (is.logical(two_datasets) && two_datasets==FALSE)) {
             return(NULL)
         }
         raw_df <- read_tsv(infile$datapath, col_types = cols())
-        
+
         original_colnames <- colnames(raw_df)
         corrected_colnames <- make.names(colnames(raw_df))
-        
-        if (!all(original_colnames == corrected_colnames)) {
-            
-            shinyalert(
-                "Input warning", 
-                paste(
-                    "Some of your column names does contain patterns which may cause downstream issues.",
-                    "Common examples are sample names starting with numbers, or containing spaces",
-                    "You will likely need to correct the following names in the data matrix for OmicLoupe to run smoothly:\n\n",
-                    paste(setdiff(original_colnames, corrected_colnames), collapse=", ")
-                ),
-                type="error")
-        }
-        
+
+        # if (!all(original_colnames == corrected_colnames)) {
```

Comment:
You can just remove this rather than commenting it out. We can always go back in the Git history if we for some reason would want to retrieve it again

Reply:
Good tip, I have removed it now!